charts/security-apps/values.yaml

```diff
@@ -161,7 +161,7 @@ vaultMonitoring:
   # -- Chart
   chart: "vault-monitoring"
   # -- [vault-monitoring Helm chart](https://github.com/adfinis/helm-charts/tree/main/charts/vault-monitoring) version
-  targetRevision: "0.3.*"
+  targetRevision: "0.4.*"
```

Comment:
can u pin the version properly? we started not doing wildcard versions a while ago, this one seems to have fallen through a crack

Reply:
Thanks, just updated it!